nailgun/entities.py

```diff
@@ -3811,6 +3811,32 @@ def list_smart_variables(self, synchronous=True, **kwargs):
         return _handle_response(response, self._server_config, synchronous)
 
 
+class Package(Entity, EntityReadMixin, EntitySearchMixin):
+    """A representation of a Package entity."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'arch': entity_fields.StringField(),
+            'checksum': entity_fields.StringField(),
+            'description': entity_fields.StringField(),
+            'epoch': entity_fields.StringField(),
+            'filename': entity_fields.StringField(),
+            'name': entity_fields.StringField(
```

Comment:
`name` would be populated for you automatically on `create`, so I was wondering what are the minimal attributes one must specify in order to create a package?